packages/backend/server/src/plugins/copilot/transcript/service.ts

```diff
@@ -49,6 +49,15 @@ export class CopilotTranscriptionService {
     private readonly providerFactory: CopilotProviderFactory
   ) {}
 
+  private async getModel(userId: string) {
+    const prompt = await this.prompt.get('Transcript audio');
+    const hasAccess = await this.models.userFeature.has(
+      userId,
+      'unlimited_copilot'
+    );
+    return prompt?.optionalModels[Number(hasAccess)];
```

Comment:
A bit hack here. Better add some comments for clearance

Reply:
done